src/reconcile/rec_visibility.c

```diff
@@ -336,9 +347,14 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
          * indicate that the value is visible to any timestamp/transaction id ahead of it.
          */
         if (upd->type == WT_UPDATE_TOMBSTONE) {
-            upd_select->stop_ts = upd->start_ts;
+            upd_select->stop_durable_ts = upd_select->stop_ts = upd->start_ts;
```

Comment:
Can you move the comment to above this line? I think this line is also related to the comment.

Reply:
Done.